mapproxy/config/loader.py

```diff
@@ -1951,13 +1951,20 @@ def tile_layers(self, grid_name_as_path=False):
                 md['format'] = self.context.caches[cache_name].image_opts().format
                 md['cache_name'] = cache_name
                 md['extent'] = extent
+                legend_version = None
+                if 'legendurl' in self.conf:
+                    wms_conf = self.context.services.conf.get('wms', None)
```

Comment:
minor: you can omit the `None` argument, as this is the default

Reply:
thanks for the hint